orienteer-core/pom.xml

```diff
@@ -10,119 +10,264 @@
+			<type>bundle</type>
+		</dependency>
+		<dependency>
+			<groupId>com.orientechnologies</groupId>
+			<artifactId>orientdb-server</artifactId>
+			<type>bundle</type>
+		</dependency>
+		<dependency>
+			<groupId>com.orientechnologies</groupId>
+			<artifactId>orientdb-client</artifactId>
+			<type>bundle</type>
+		</dependency>
+		<dependency>
+			<groupId>com.google.inject</groupId>
+			<artifactId>guice</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.google.inject.extensions</groupId>
+			<artifactId>guice-servlet</artifactId>
+		</dependency>
 		<dependency>
 			<groupId>org.apache.tika</groupId>
 			<artifactId>tika-core</artifactId>
 			<version>1.13</version>
 		</dependency>
 
 		<!-- WEB JARS -->
-	    <dependency>
-		    <groupId>de.agilecoders.wicket.webjars</groupId>
-		    <artifactId>wicket-webjars</artifactId>
-	    </dependency>
-	    <dependency>
+		<dependency>
+			<groupId>de.agilecoders.wicket.webjars</groupId>
+			<artifactId>wicket-webjars</artifactId>
+		</dependency>
+		<dependency>
 			<groupId>org.webjars</groupId>
 			<artifactId>font-awesome</artifactId>
-	    </dependency>
-	    <dependency>
+		</dependency>
+		<dependency>
 			<groupId>org.webjars</groupId>
 			<artifactId>bootstrap</artifactId>
-	    </dependency>
-	    <dependency>
-            <groupId>org.webjars</groupId>
-            <artifactId>gridster.js</artifactId>
-        </dependency>
-        <dependency>
-		    <groupId>org.webjars</groupId>
-		    <artifactId>jquery-ui</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.webjars</groupId>
+			<artifactId>gridster.js</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.webjars</groupId>
+			<artifactId>jquery-ui</artifactId>
 		</dependency>
 		<dependency>
 			<groupId>org.webjars</groupId>
 			<artifactId>metisMenu</artifactId>
 		</dependency>
-	    <!-- LOGGING DEPENDENCIES - LOG4J -->
-	    <dependency>
-		    <groupId>org.slf4j</groupId>
-		    <artifactId>slf4j-log4j12</artifactId>
-	    </dependency>
-	    <dependency>
-		    <groupId>log4j</groupId>
-		    <artifactId>log4j</artifactId>
-	    </dependency>
 
-	    <!--  JUNIT DEPENDENCY FOR TESTING -->
-	    <dependency>
-		    <groupId>junit</groupId>
-		    <artifactId>junit</artifactId>
-		    <scope>test</scope>
-	    </dependency>
-	    <dependency>
-		<groupId>javax.servlet</groupId>
-		<artifactId>javax.servlet-api</artifactId>
-	    </dependency>
-        <!-- MARKDOWN DEPENDENCY -->
-        <!--<dependency>
+		<!-- CLASS LOADING -->
+		<dependency>
+			<groupId>org.kevoree.kcl</groupId>
+			<artifactId>org.kevoree.kcl</artifactId>
+			<version>5</version>
+		</dependency>
+		<!-- AETHER DOWNLOAD MAVEN DEPENDENCIES START-->
+		<dependency>
+			<groupId>org.apache.maven</groupId>
+			<artifactId>maven-aether-provider</artifactId>
+			<version>${mavendep.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.maven</groupId>
+			<artifactId>maven-settings-builder</artifactId>
```

Comment:
How this is being used? If it's dependency of some other lib - can it be loaded recursively?